packages/mobile/src/screens/sign-up-screen/screens/SelectArtistsScreen.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+
+import { getGenres } from 'audius-client/src/common/store/pages/signon/selectors'
+import { Pressable, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import { Text } from 'app/components/core'
+
+const messages = {
+  header: 'Follow At Least 3 Artists',
+  description:
+    'Curate your feed with tracks uploaded or reposted by anyone you follow. Click the artist’s photo to preview their music.',
+  continue: 'Continue'
+}
+
+export const SelectArtistsScreen = () => {
+  const genres = useSelector((state: any) => ['Featured', ...getGenres(state)])
+  const [currentGenre, setCurrentGenre] = useState('Featured')
+
+  return (
+    <View>
+      <View>
+        <Text>{messages.header}</Text>
+        <Text>{messages.description}</Text>
+      </View>
+      <View accessibilityRole='radiogroup'>
+        {genres.map((genre) => {
+          const checked = genre === currentGenre
+          return (
+            <Pressable
+              key={genre}
+              testID={genre}
+              accessibilityRole='radio'
+              accessibilityState={{ checked }}
+              accessibilityLiveRegion='polite'
+              onPress={() => setCurrentGenre(genre)}
+              style={{ backgroundColor: checked ? 'purple' : undefined }}
```

Comment:
do we have a color var for this purple?

Reply:
Yeah all that is to be turned into a real component, this was for quick visual aide to make sure things were actually being selected. My approach here is build the minumun accessible field and then spend the time reimplementing selectable-pill to work with these accessibility constraints. It was getting hairy approaching that refactor at this point